src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import "./Navbar.scss"
+
+
+function Navbar() {
+  return (
+    <div className="Navbar">
+         <div className="navbar__left">
```

Comment:
className here should also start with capital N (Navbar__left, Navbar__right, Navbar__right__link,  Navbar__right__btn etc)